test/suites/static_analysis.sh

```diff
@@ -68,9 +68,7 @@ test_static_analysis() {
       golint -set_exit_status lxd-p2c
 
       golint -set_exit_status lxd/config
-      golint -set_exit_status lxd/db/node
-      golint -set_exit_status lxd/db/query
-      golint -set_exit_status lxd/db/schema
```

Comment:
You shouldn't remove the sub-packages, golint isn't recursive, it only evaluates one package.

Reply:
Well, it can be made recursive but then it'd be "lxd/db/..." (untested).

Reply:
Fixed.